Cargo.toml

```diff
@@ -19,7 +15,7 @@ repository = "https://github.com/AscendingCreations/AxumCSRF"
 
 [features]
 default = []
-layer = ["tower-layer", "tower-service"]
+layer = ["dep:tower-layer", "dep:tower-service"]
```

Comment:
This should remove tower-layer and tower-service being listed as a feature, where-as it's just an optional dep.